tests/jest-extensions-babel-transform.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+const babelJest = require( 'babel-jest' );
+
+module.exports = babelJest.createTransformer( {
+	presets: [
+		require.resolve( '@automattic/calypso-build/babel/default' ),
+		require.resolve( '@automattic/calypso-build/babel/wordpress-element' ),
```

Comment:
The `@automattic/calypso-build` preset provides this transform _without_ this line.

That is something the preset should provide, but I'm not sure of the best way to go about it.

This aligns with the way we pass `--env.WP` to webpack, but Jest doesn't seem to have any concept of arbitrary arguments we pass through the CLI. In fact, jest fails if it doesn't understand the arguments passed to the CLI.

This is sufficient for now, but it's something that should be considered for future calypso-build work.

Reply:
Good point, yep :+1: